Cargo.toml

```diff
@@ -2,5 +2,14 @@
 members = ["spotify_player", "lyric_finder"]
 resolver = "2"
 
+[workspace.package]
+rust-version = "1.74"
+
+[profile.ci]
```

Comment:
what is `ci` profile for? How is it different from `dev`?

Reply:
Setting `debug = 0` avoids emitting a large amount of debug information which is largely not needed when just checking if the crate can compile. That way the `./target/` cache is smaller. Likewise with `incremental` and `strip`. I was mostly following [matklad's recommendations on the topic](https://matklad.github.io/2021/09/04/fast-rust-builds.html#CI-Workflow). 

I admittedly am not sure about `incremental`, as it does reduce the size but I do not know about its performance implications, which in any case would be limited to compiling the spotify_player and lyric_finder crates as the dependencies are all cached.

For a ballpark estimation, I just run `cargo check` (debug profile) and `cargo check --profile ci` in my Linux machine to compare the sizes of the output: the former was 665M, the latter 367M.

Reply:
got it, make sense to me. Thanks for the context